src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component } from 'react';
+import './MovieDetails.css';
+import { Movie } from '../../models';
+
+export interface MovieDetailsProps {
+    movie: Movie | undefined;
+}
+
+export default class MovieDetails extends Component<MovieDetailsProps> {
+    getMovieDuration(mins: number) {
+        let hours = Math.floor(mins / 60);
+        let minutes = mins % 60;
+        return `${hours}h ${minutes}min`;
+    }
+
+    getYear(mins: number) {
```

Comment:
Optional. Duplicate method. Name is different but body is the same as getMovieDuration.

Reply:
will be fixed in the next module